lecli/lecli.py

```diff
@@ -4,9 +4,10 @@
 import queryapi
 import userapi
 
-
+VERSION = 0.1
+apiutils.VERSION = VERSION
```

Comment:
Why don't we store version number in one place? Importing a module containing version (like _version.py) won't do any harm to modularity.

I like the way its implemented in twisted:
https://github.com/powdahound/twisted/blob/master/twisted/_version.py
https://github.com/powdahound/twisted/blob/master/twisted/__init__.py